src/community/community.ts

```diff
@@ -146,18 +146,21 @@ export class Community extends EventEmitter {
 }
 
 /**
- * This waits until the libp2p node has connected to two peers
+ * This waits until the libp2p node has connected to a signer
  * @private
  */
-export function afterThreePeersConnected(node:IP2PNode):Promise<void> {
+export async function afterOneSignerConnected(node:IP2PNode, group:NotaryGroup):Promise<void> {
+    const peerIds = await notaryGroupToSignerPeerIds(group) 
     return new Promise((resolve) => {
-        let connectCount = 0
-        const onConnect = async ()=> {
-            debugLog("peer connected: ", connectCount)
-            connectCount++
-            if (connectCount >= 3) {
-                node.off('peer:connect', onConnect)
-                resolve()
+        const onConnect = (peer:any)=> {
+            debugLog("peer connected: ", peer.id.toB58String())
+            for (let id of peerIds) {
+                if (id.isEqual(peer.id)) {
```

Comment:
Can the previous two lines just be `peerIds.incldues(peer.id)`?

Reply:
I don't think so because they are different objects? it's a PeerId not a string.

Reply:
Ah definitely, sorry totally read it as a string 👍